content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -645,7 +656,7 @@ This determines whether a user can modify the mode of the source volume when a
 {{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}} is being
 created from a VolumeSnapshot.
 
-Refer to [Converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode) 
+Refer to [Converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode)
```

Comment:
It looks like your text editor has removed some whitespace here. Watch out for this - tidying up like this is best done in its own commit.

Reply:
okay, so is there anything I can do regarding this?